Marlin/Conditionals_LCD.h

```diff
@@ -108,7 +108,26 @@
     #define ULTIPANEL
     #define NEWPANEL
 
-  #endif
+  #elif ENABLED(CR10_stockdisplay)
+  
+    #define REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
+    #define ST7920_DELAY_1 DELAY_2_NOP
+    #define ST7920_DELAY_2 DELAY_2_NOP
+    #define ST7920_DELAY_3 DELAY_2_NOP
+
+  #elif ENABLED (MKS_OLED13_128x64_FULL_GRAPHICS_CONTROLLER) 
+  
+    #define U8GLIB_SH1106 
+    #define REPRAP_DISCOUNT_SMART_CONTROLLER 
+    #define NEWPANEL 
+    #define REVERSE_MENU_DIRECTION
```

Comment:
Won't this override in all circumstances for this display ?  Doesn't this need to honor REVERSE_MENU_DIRECTION if set in Configuration.h ?

Reply:
I was looking at something else and saw this, so I think your `REVERSE_MENU_DIRECTION` might over-rided by `Configuration.h` without including any ifdef/ifenabled checks (although it mentions only `pins.h` and `Configuration_adv.h`):

```
/**
 * Conditionals_LCD.h
 * Conditionals that need to be set before Configuration_adv.h or pins.h
 */

#ifndef CONDITIONALS_LCD_H // Get the LCD defines which are needed first
```

To be 100% sure, just put your changes into `Conditionals_LCD.h`, then compile / load on your printer with and without `REVERSE_MENU_DIRECTION` set in `Configuration.h` and ensure it changes.  If it does, you should be good so no changes would be needed.